### ISSUE TITLE
Improve update experience

### ISSUE THREAD

**Issue by piotrski** [elk-zone/elk-native#42]
The current update alert is okay, but it could be improved.
- Include release notes with markdown formatting
- It would be beneficial to add information about the authors and group the changes similarly to how [elk](https://github.com/elk-zone/elk/releases/) does.

![image](https://user-images.githubusercontent.com/244174/213482303-55573c39-8a7e-4024-be40-36814a04af64.png)

**Comment by JonasKruckenberg** [comment]
Yes, this is the default built-in dialog, just for getting something up and running, I imagine it to be a proper HTML dialog, but that off requires adding it to elk proper (and then we could support md formatting)

> It would be beneficial to add information about the authors and group the changes similarly to how [elk](https://github.com/elk-zone/elk/releases/) does.

I think we can change the template for the release notes, no idea about attributing authors though. I'm gonna ping @jbolda since he's the expert behind convector. For example I don't really like the links to commits in this situation, they don't really add value to end-users.

**Comment by jbolda** [comment]
For reference, this is what it looks like on windows.
![image](https://user-images.githubusercontent.com/2019387/213784321-d272a82e-7976-4e34-b1d1-90b56102c49f.png)

**Comment by piotrski** [comment]
@jbolda should I assign this to you?

**Comment by JonasKruckenberg** [comment]
Hey! Just letting you know that elk-native is [no longer maintained](https://github.com/elk-zone/elk-native/pull/96/), much of the team has moved on or is busy with other projects (there's only so much time in a day 😅), so this issue will likely not be fixed.
Please use the actively maintained elk PWA instead.

Thanks so much for your support and understanding so far! ❤️
